procurement/filters.py

```diff
@@ -7,21 +8,27 @@
     InternalServerErrorException,
     ForbiddenException,
 )
-from procurement.models import Classification, Tender
+from procurement.models import Classification, Council, Tender
 
 class TenderFilter(filters.FilterSet):
     awards__end_date = filters.DateFromToRangeFilter()
     classification = filters.ModelMultipleChoiceFilter(
         field_name="tenderclassification__classification__group",
         queryset=Classification.objects.all().distinct("group"),
         to_field_name="group",
+        widget=forms.CheckboxSelectMultiple()
     )
 
     pc = filters.CharFilter(
         field_name="council__gss_code",
         method="filter_postcode",
     )
 
+    council = filters.ModelMultipleChoiceFilter(
```

Comment:
I wonder if as this filter is being used in two places with slightly different sets of fields it might be tidier to have a Base filter and then have one filter class for the front page/council page and one for the alerts page that inherit from the common one? I don't think it makes much difference functionality wise at the moment.

Reply:
This has now been done!